blocks/library/button/index.js

```diff
@@ -115,18 +115,18 @@ registerBlockType( 'core/button', {
 							checked={ !! clear }
 							onChange={ toggleClear }
 						/>
-						<PanelBody title={ __( 'Button Background Color' ) }>
+						<PanelColor title={ __( 'Button Background' ) } colorValue={ color } key="panel-color-button-1">
```

Comment:
I'd recomment to don't remove the word "Color". See below. Instead, I'd consider to remove the word "Button", for consistency with the paragraphs and because it's already clear we're editing a button block.